commands/list/nodepools/command.go

```diff
@@ -0,0 +1,206 @@
+// Package nodepools implements the 'list organizations' sub-command.
+package nodepools
+
+import (
+	"fmt"
+	"os"
+	"sort"
+	"strconv"
+	"strings"
+
+	"github.com/fatih/color"
+	"github.com/giantswarm/columnize"
+	"github.com/giantswarm/gsclientgen/models"
+	"github.com/giantswarm/microerror"
+	"github.com/spf13/cobra"
+
+	"github.com/giantswarm/gsctl/client"
+	"github.com/giantswarm/gsctl/client/clienterror"
+	"github.com/giantswarm/gsctl/commands/errors"
+	"github.com/giantswarm/gsctl/config"
+	"github.com/giantswarm/gsctl/flags"
+	"github.com/giantswarm/gsctl/nodespec"
+)
+
+var (
+	// Command performs the "list organizations" function
+	Command = &cobra.Command{
+		Use:     "nodepools <cluster-id>",
```

Comment:
When listing keypairs, we pass the cluster ID in as a flag (https://github.com/giantswarm/gsctl/blob/master/commands/list/keypairs/command.go#L61), I personally feel that's cleaner and we should stay consistent here

Reply:
We are actually gradually moving towards positional arguments everywhere in gsctl when an option is mandatory.

Reply:
OK, then that makes sense